src/main/java/application/FormController.java

```diff
@@ -341,7 +349,9 @@ public FormEntryResponseBean getNext(@RequestBody SessionRequestBean requestBean
                     formSession.getFormEntryModel(),
                     formSession.getCurrentIndex());
             updateSession(formSession, serializableFormSession);
-            FormEntryResponseBean responseBean = mapper.readValue(resp.toString(), FormEntryResponseBean.class);
+            FormEntryNavigationResponseBean responseBean = mapper.readValue(resp.toString(), FormEntryNavigationResponseBean.class);
+            responseBean.setIsAtLastIndex(formSession.getIsAtLastIndex());
```

Comment:
Do you want me to change the `getCurrentJson` signature to accept additional params just for this use case? seems like more of a hassle than using it only where it's needed